addon/usables/use-machine.ts

```diff
@@ -27,20 +25,29 @@ const ERROR_CANT_RECONFIGURE = `Cannot re-invoke withContext after the interpret
 const ERROR_CHART_MISSING = `A statechart was not passed`;
 
 type Args<Context, Schema extends StateSchema, Event extends EventObject> = {
-  positional?: [MachineConfig<Context, Schema, Event>];
-  named?: {
-    chart: MachineConfig<Context, Schema, Event>;
-    config?: Partial<MachineOptions<Context, Event>>;
-    context?: Context;
-    initialState?: State<Context, Event>;
-    onTransition?: StateListener<Context, Event, Schema, Typestate<Context>>;
-  };
+  positional?: [
```

Comment:
I don't know if we want this change -- does the args this way means we can't as easily switch back to `.withContext()` like APIs when we no longer rely on a build-time transform for Resources to work

Reply:
@NullVoxPopuli Great question -- I'm not sure. I wouldn't think this would change anything (compared to use `named`). I think it's all how we handle these things in `setup` depends on if we're using the `withContext` API.